Task_TheGridProblem/index.html

```diff
@@ -1,11 +1,23 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <title>Grid Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
```

Comment:
The 'index.css' file should be properly linked in the 'link' tag with the correct file path.